load-generator/src/main/java/com/amazon/opentelemetry/load/generator/emitter/OtlpMetricEmitter.java

```diff
@@ -16,23 +16,24 @@
 package com.amazon.opentelemetry.load.generator.emitter;
 
 import com.amazon.opentelemetry.load.generator.model.Parameter;
-import io.grpc.ManagedChannelBuilder;
-import io.opentelemetry.api.metrics.GlobalMeterProvider;
-import io.opentelemetry.api.metrics.common.Labels;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.DoubleHistogram;
 import io.opentelemetry.api.metrics.LongCounter;
-import io.opentelemetry.api.metrics.LongValueRecorder;
 import io.opentelemetry.api.metrics.Meter;
 import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter;
 import io.opentelemetry.sdk.metrics.SdkMeterProvider;
-import io.opentelemetry.sdk.metrics.export.IntervalMetricReader;
 import io.opentelemetry.sdk.metrics.export.MetricExporter;
-import java.util.Collections;
+import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
 
 public class OtlpMetricEmitter extends MetricEmitter {
 
   private final static String API_NAME = "loadTest";
   LongCounter apiBytesSentCounter;
-  LongValueRecorder apiLatencyRecorder;
```

Comment:
Long value recorder is no longer supported. Closest I found was DoubleHistogram

Reply:
There is a long histogram but Meter only has double histogram

Reply:
There was a migration guide from 1.4.0 -> 1.5.0 https://github.com/open-telemetry/opentelemetry-java/discussions/3506
I'm not sure how much it still applies, but they used the `LongHistogram` to replace the `LongValueRecorder`.